src/Factory.php

```diff
@@ -203,7 +203,6 @@ protected function _factory($seed, $defaults = []): object
 
             $seed = $this->_mergeSeeds($seed, $defaults);
         }
-        unset($defaults);
```

Comment:
no :)

Reply:
you will not use that $defaults further and is not by reference.

Reply:
yes, but for explicitly stating that we do not want any use in the future :)